server/Cargo.toml

```diff
@@ -17,7 +17,7 @@ thiserror = "1.0.30"
 actix-rt = "2.6.0"
 assert-json-diff = "2.0.1"
 async-std = "1.12.0"
-
+rand = "0.8.5"
```

Comment:
Added this so that can shuffle array in temperature chart repo tests (to test order by, by inserting in temperature logs in random order), it was used in another crate, moved to workspace